BUILD.bazel

```diff
@@ -0,0 +1,211 @@
+load("//bazel:swift_cc_defs.bzl", "UNIT", "swift_cc_library", "swift_cc_test")
+load("//:copts.bzl", "COPTS")
+
+swift_cc_library(
```

Comment:
Sorry, one last thing.

Only use the `swift_cc` wrappers for code that's using our custom swift cmake wrappers.

Albatross is just using regular cmake `add_library`.

It probably doesn't make much of a difference here, but we'll likely be unable to compile code that's not already using those wrappers in other projects (it doesn't respect all the warnings/settings).

Reply:
Done